import_email_events.py

```diff
@@ -0,0 +1,24 @@
+#
+# Script to import "email event" metrics from S3 into redshift.
+#
+
+import import_events
+SCHEMA = """
+    type VARCHAR(30) NOT NULL ENCODE lzo,
+    locale VARCHAR(40) ENCODE lzo,
+    domain VARCHAR(40) ENCODE lzo,
+    template VARCHAR(64) ENCODE lzo,
+    flow_id VARCHAR(64) DISTKEY ENCODE lzo,
+    complaint VARCHAR(64) ENCODE lzo,
+    bounced VARCHAR(64) ENCODE lzo
+"""
+
+COLUMNS = "type, locale, domain, template, flow_id, complaint, bounced"
```

Comment:
I think the order here is still not right? Looking at an example line from the CSV, `flow_id` should be first, `domain` should be second, `locale` should be last...